src/api/java/com/minecolonies/api/colony/colonyEvents/descriptions/IColonyEventDescription.java

```diff
@@ -0,0 +1,79 @@
+package com.minecolonies.api.colony.colonyEvents.descriptions;
+
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * Description for an event that happened in the colony.
+ */
+public interface IColonyEventDescription
```

Comment:
`extends INBTSerializable<CompoundNBT>` should be used here instead of the custom methods.

Reply:
done.

Reply:
This is now also done of `IColonyEvent`.